test/jit/test_save_load.py

```diff
@@ -70,6 +70,394 @@ def current_subcmul(a, b, alpha=1):
         self.assertEqual(no_alpha_current, current_subcmul(a, b))
         self.assertEqual(with_alpha_current, current_subcmul(*args))
 
+    # Helper that returns the module after saving and loading
+    def _save_load_module(self, m):
+        scripted_module = torch.jit.script(m())
+        buffer = io.BytesIO()
+        torch.jit.save(scripted_module, buffer)
+        buffer.seek(0)
+        return torch.jit.load(buffer)
+
+    # Helper which returns the result of a function or the exception the
+    #   function threw.
+    def _try_fn(self, fn, *args, **kwargs):
+        result = None
+        try:
+            result = fn(*args, **kwargs)
```

Comment:
nit: maybe a bit simpler to return in the try and return the exception?  Then you don't need a bogus assignment to result at the beginning.

Reply:
Sure. Sounds good.